pkg/collector/corechecks/cluster/kubeapi/kubernetes_topology_config.go

```diff
@@ -19,22 +20,29 @@ const (
 	Kubernetes ClusterType = "kubernetes"
 	// OpenShift cluster type
 	OpenShift = "openshift"
+
```

Comment:
Maybe golint will complain about a comment here

Reply:
I hope it won't, because it's legit grouping